Stanford/JumpstartVPSA/Install/JSVPSAFinishInstallation.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * @file
+ * Abstract Task Class.
+ */
+
+namespace Stanford\JumpstartVPSA\Install;
+/**
+ *
+ */
+class JSVPSAFinishInstallation extends \AbstractInstallTask {
+
+  /**
+   * Enable a couple of final modules because they depend on everything.
+   *
+   * @param array $args
+   *   Installation arguments.
+   */
+  public function execute(&$args = array()) {
+
+    // Set the homepage to the home node.
+    $home_node_path = drupal_get_normal_path('home');
+    variable_set('site_frontpage', $home_node_path);
+    // Oh man so many static variables to kill!
+    static $ordered;
+    static $all_links;
+    static $cache;
+    static $drupal_static_cache;
+    $ordered = null;
+    $all_links = null;
+    $cache = null;
+    $drupal_static_cache = null;
+    // Call flush all caches so that the blocks table gets populated.
+    drupal_static_reset();
+    drupal_flush_all_caches();
+    // Revert dem all!
+    // features_rebuild();
+    // features_revert();
+    features_revert('stanford_video');
+    // Small fix for a menu bug
+    db_update('menu_links')
+      ->fields(array(
+        'router_path' => 'admin/manage'
+      ))
+      ->condition('link_path', 'admin/manage')
+      ->execute();
+    // Add some block titles.
+    // module, delta, title
+    $blocks = array(
```

Comment:
This seems like it should belong in the content section but I guess that it depends on the cache clears. Good here for now unless you know better.